src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.xaml

```diff
@@ -0,0 +1,306 @@
+											Storyboard.TargetName="ScrollBarSeparator"
+											Storyboard.TargetProperty="Opacity"
+											To="1" />
+									</Storyboard>
+								</VisualState>
+								<VisualState x:Name="ScrollBarSeparatorExpandedWithoutAnimation">
+									<Storyboard>
+										<DoubleAnimation
+											Duration="0"
+											BeginTime="{ThemeResource ScrollViewerSeparatorExpandBeginTime}"
+											Storyboard.TargetName="ScrollBarSeparator"
+											Storyboard.TargetProperty="Opacity"
+											To="1" />
+									</Storyboard>
+								</VisualState>
+								<VisualState x:Name="ScrollBarSeparatorCollapsedWithoutAnimation">
+									<Storyboard>
+										<DoubleAnimation
+											Duration="0"
+											BeginTime="{ThemeResource ScrollViewerSeparatorContractBeginTime}"
+											Storyboard.TargetName="ScrollBarSeparator"
+											Storyboard.TargetProperty="Opacity"
+											To="0" />
+									</Storyboard>
+								</VisualState>
+							</VisualStateGroup>
+						</netstdref:VisualStateManager.VisualStateGroups>
+
+						<Grid Background="{TemplateBinding Background}" x:Name="Root">
+							<netstdref:Grid.ColumnDefinitions>
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</netstdref:Grid.ColumnDefinitions>
+							<netstdref:Grid.RowDefinitions>
+								<RowDefinition Height="*" />
+								<RowDefinition Height="Auto" />
+							</netstdref:Grid.RowDefinitions>
+							<!-- ContentTemplate not yet supported for ScrollContentPresenter-->
+							<ScrollContentPresenter
+								x:Name="ScrollContentPresenter"
+								netstdref:Grid.RowSpan="2"
+								netstdref:Grid.ColumnSpan="2"
+								Margin="{TemplateBinding Padding}" />
+
+							<netstdref:ScrollBar
+								x:Name="VerticalScrollBar"
+								Grid.Column="1"
+								IsTabStop="False"
+								Maximum="{TemplateBinding ScrollableHeight}"
+								Orientation="Vertical"
+								Visibility="{TemplateBinding ComputedVerticalScrollBarVisibility}"
+								Value="{TemplateBinding VerticalOffset}"
+								ViewportSize="{TemplateBinding ViewportHeight}"
+								HorizontalAlignment="Right"
+								skia:Width="16"/>
+
+							<netstdref:ScrollBar
+								x:Name="HorizontalScrollBar"
+								IsTabStop="False"
+								Maximum="{TemplateBinding ScrollableWidth}"
+								Orientation="Horizontal"
+								Grid.Row="1"
+								Visibility="{TemplateBinding ComputedHorizontalScrollBarVisibility}"
+								Value="{TemplateBinding HorizontalOffset}"
+								ViewportSize="{TemplateBinding ViewportWidth}"
+								skia:Height="16"/>
+
+							<netstdref:Border
+								x:Name="ScrollBarSeparator"
+								Grid.Row="1"
+								Grid.Column="1"
+								Opacity="0"
+								Background="{ThemeResource SystemControlPageBackgroundChromeLowBrush}" />
+						</Grid>
+					</Border>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+
+	<Style
```

Comment:
Would be good to have a native style variant that uses the native scrollbars.

Reply:
Done `NativeScrollViewerStyle`